Code/ForceField/AngleConstraint.h

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright (C) 2004-2024 Paolo Tosco and other RDKit contributors
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include <RDGeneral/export.h>
+#ifndef __RD_ANGLECONSTRAINT_H__
+#define __RD_ANGLECONSTRAINT_H__
```

Comment:
We used to prefix these with `__`, but it's considered bad style, so I change them whenever I notice
```suggestion
#ifndef RD_ANGLECONSTRAINT_H
#define RD_ANGLECONSTRAINT_H
```